### PR TITLE
vfep-377 create a new field relationshipAndChildType in the FE as per  the recommendation.

### PR DESCRIPTION
**Pull request by GovNapoleon**
## Summary

- vfep-377 create a new field relationshipAndChildType in the FE as per  the recommendation.

## JIRA
  
   [VFEP-377](https://vajira.max.gov/browse/VFEP-377)

## Testing done

- Looked at the review instance and verified that everything was still working correctly.

## Screenshots
_Note: This field is mandatory for component work and UI changes (non-component work should NOT have screenshots)._

| | Before | After |
| --- | --- | --- |
| Mobile | | |
| Desktop | | |

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature
- [ ]  [Accessibility foundational testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by jsimonVA — APPROVED** [review]
LGTM, the unit test seems to need adjusted.

File: src/applications/edu-benefits/5490/tests/config/applicantInformation.unit.spec.jsx
Comment (rmessina1010) [inline src/applications/edu-benefits/5490/tests/config/applicantInformation.unit.spec.jsx:1]: This test has an error,  could we try updating this line to

`expect(formDOM.querySelectorAll('input, select').length).to.equal(12); 
`


</body>
</html>

File: src/applications/edu-benefits/5490/tests/config/sponsorInformation.unit.spec.jsx
Comment (rmessina1010) [inline src/applications/edu-benefits/5490/tests/config/sponsorInformation.unit.spec.jsx:1]: Another unit test fail, `input#root_spouseInfo_divorcePendingYes` is null (not foud)

File: src/applications/edu-benefits/5490/tests/config/applicantInformation.unit.spec.jsx
Comment (GcioNapoleon) [reply src/applications/edu-benefits/5490/tests/config/applicantInformation.unit.spec.jsx:1]: I fixed this one

File: src/applications/edu-benefits/5490/tests/config/sponsorInformation.unit.spec.jsx
Comment (GcioNapoleon) [reply src/applications/edu-benefits/5490/tests/config/sponsorInformation.unit.spec.jsx:1]: I don't see this after I fixed the other one

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**